evm/utils/test_builder/README.md

```diff
@@ -0,0 +1,183 @@
+# JSON test builder
+
+
+The JSON test builder found in `evm.utils.test_builder.test_builder` is a library that
```

Comment:
I recently introduced `evm.tools` into the module.  I think that `test_builder` should probably be moved there.